api/routes/slide.js

```diff
@@ -14,8 +19,59 @@ router
       .catch(err => next(err))
   })
   .post('/', (req, res, next) => {
-    const { title, description, duration, slideshow } = req.body
-    // TODO(@alexander-e-andrews) Write code for adding a slide to the database
+    try {
+      var newSlide = new Slide()
+      var slideshowID
+      /**
+       * Try CATCH error block
+       * If front end does not send all data, its okay
+       * Only requires a parent slideshow
+       * Catches do nothing, as they are left blank
+       */
+      try {
```

Comment:
I'm not sure about all these try catch blocks. What are they needed for?

Reply:
Same concern, I'd rather use `if (typeof req.body.data !== 'undefined')`